docs/source/datasets/index.rst

```diff
@@ -304,9 +309,11 @@ The Social Security Administration (SSA) is the US federal government agency tha
 that consists of retirement, disability and survivor benefits. To find out more about this program, visit the `SSA homepage <https://www.ssa.gov/about-ssa>`_.
 
 pseudopeople can generate a simulated version of a subset of the administrative data that would be recorded by SSA.
+Currently, the simulated SSA data includes records of SSA creation and dates of death.
 This is a yearly data file that is **cumulative** -- when you specify a year, you will recieve all records *up to the end of*
 that year.
-Currently, the simulated SSA data includes records of SSA creation and dates of death.
+The simulated SSA data files will only include records about simulants who were alive during our simulated timespan of 2019-2041.
+Therefore, while SSA data files can be generated for years prior to 2019, they will only include records for SSA creation.
```

Comment:
When I first read this it seemed contradictory -- it was unclear how the final clause followed from the previous sentence, because it seemed like you meant it would include _all_ creation events in earlier years. Perhaps say something like: "Therefore, while SSA data files can be generated for years prior to 2019, they will only include records for SSA creation for simulants who were still alive during the simulation." Except that seems a bit redundant with the previous sentence.